### PR TITLE
Condensed Features Experiment

### PR DESCRIPTION
**Pull request by niranjan-uma-shankar**
#### Proposed Changes

* Implements an A/B test on the signup plans step.
* The changes here try out a different visual treatment of plan features.
* Read pbxNRc-1WS for more information.

**Annual**

Screenshot 2022-09-01 at 6 03 45 PM

**Monthly**

Screenshot 2022-09-01 at 6 03 54 PM

#### Testing Instructions

 
* Assign yourself to treatment of the experiment `calypso_signup_plans_step_optimize_202208_v1` in Abacus.
* Signup for a new account and verify that the plans step UI matches the one shown in screenshots above.
* Verify that mobile view is unchanged - this experiment is only for desktop, so mobile view should have no UI changes.
* Verify that non-EN is unaffected
* Verify that control version in unaffacted.
* Verify that you can continue to complete signup and create a site / complete a purchase.

### CODE REVIEW

File: client/my-sites/plan-features-comparison/index.jsx [hunk 1]
```diff
@@ -319,6 +335,72 @@ export class PlanFeaturesComparison extends Component {
...
+			const classes = classNames( '', getPlanClass( planName ), {
+				'is-last-feature': featureIndex + 1 === features.length,
+				'is-highlighted':
+					selectedFeature && currentFeature && selectedFeature === currentFeature.getSlug(),
+				'is-bold': currentFeature.getSlug() === FEATURE_CUSTOM_DOMAIN,
+			} );
+
+			if ( currentFeature.isUniqueFeature ) {
+				return (
+					<li className={ classes }>{ this.renderFeatureItem( currentFeature, mapIndex ) }</li>
```
Comment (claudiucelfilip) [inline client/my-sites/plan-features-comparison/index.jsx]: I'm getting a warning about missing "key" prop. We should probably add it to the `li` and maybe to the `div` below.

**Review by claudiucelfilip — APPROVED** [review]
Works as described.
LGTM

File: client/lib/plans/features-list.js [hunk 2]
```diff
@@ -1747,6 +1761,70 @@ export const FEATURES_LIST = {
...
+		getDescription: () => {},
+	},
+	[ FEATURE_PREMIUM_THEMES_TEST ]: {
+		getSlug: () => FEATURE_PREMIUM_THEMES_TEST,
+		getTitle: () => 'Unlimited premium themes for your site',
+		getDescription: () => {},
+	},
+	[ FEATURE_PRIORITY_SUPPORT_TEST ]: {
+		getSlug: () => FEATURE_PRIORITY_SUPPORT_TEST,
+		getTitle: () => 'Priority chat and email support, 24X7.',
```
Comment (donlair) [inline client/lib/plans/features-list.js]: Change to `24/7 live chat support`

File: client/lib/plans/features-list.js [hunk 3]
```diff
@@ -1747,6 +1761,70 @@ export const FEATURES_LIST = {
...
+		getDescription: () => {},
+	},
+	[ FEATURE_PLUGINS_TEST ]: {
+		getSlug: () => FEATURE_PLUGINS_TEST,
+		getTitle: () =>
+			'Install WordPress Plugins. Choose among thousands of plugins to easily add forms, bookings, or custom fields.',
```
Comment (donlair) [inline client/lib/plans/features-list.js]: Change to `Install WordPress plugins`

File: client/lib/plans/features-list.js [hunk 4]
```diff
@@ -1747,6 +1761,70 @@ export const FEATURES_LIST = {
...
+
+	/* START - condensed_plan_features_v1 test */
+	[ FEATURE_HOSTING_TEST ]: {
+		getSlug: () => FEATURE_HOSTING_TEST,
+		getTitle: () => i18n.translate( 'Fully managed web hosting & CDN' ),
+		getDescription: () => {},
+	},
+	[ FEATURE_PREMIUM_THEMES_TEST ]: {
+		getSlug: () => FEATURE_PREMIUM_THEMES_TEST,
+		getTitle: () => 'Unlimited premium themes for your site',
```
Comment (donlair) [inline client/lib/plans/features-list.js]: Change to `Unlimited premium themes`

File: client/lib/plans/features-list.js [hunk 5]
```diff
@@ -1747,6 +1761,70 @@ export const FEATURES_LIST = {
...
+	},
+	[ FEATURE_PLUGINS_TEST ]: {
+		getSlug: () => FEATURE_PLUGINS_TEST,
+		getTitle: () =>
+			'Install WordPress Plugins. Choose among thousands of plugins to easily add forms, bookings, or custom fields.',
+		getDescription: () => {},
+	},
+	[ FEATURE_SFTP_DATABASE_TEST ]: {
+		getSlug: () => FEATURE_SFTP_DATABASE_TEST,
+		getTitle: () => 'SFTP (Secure File Transfer Protocol) and Database access.',
```
Comment (donlair) [inline client/lib/plans/features-list.js]: Change to `SFTP and Database Access`

File: client/lib/plans/features-list.js [hunk 6]
```diff
@@ -1747,6 +1761,70 @@ export const FEATURES_LIST = {
...
+		getDescription: () => {},
+	},
+	[ FEATURE_PAID_NEWSLETTER_V1 ]: {
+		getSlug: () => FEATURE_PAID_NEWSLETTER_V1,
+		getTitle: () => 'Paid newsletter w/ unlimited subscribers.',
+		getDescription: () => {},
+	},
+	[ FEATURE_REPUBLICIZE_V3 ]: {
+		getSlug: () => FEATURE_REPUBLICIZE,
+		getTitle: () => 'Advanced social media marketing tools',
```
Comment (donlair) [inline client/lib/plans/features-list.js]: Change to `Advanced social media tools`

File: client/lib/plans/features-list.js [hunk 7]
```diff
@@ -1747,6 +1761,70 @@ export const FEATURES_LIST = {
...
+		getDescription: () => {},
+	},
+	[ FEATURE_MONETISE_V2 ]: {
+		getSlug: () => FEATURE_MONETISE,
+		getTitle: () => 'Monetize your website with ads',
```
Comment (donlair) [inline client/lib/plans/features-list.js]: Change to `Earn money from ads`

File: client/lib/plans/features-list.js [hunk 8]
```diff
@@ -1747,6 +1761,70 @@ export const FEATURES_LIST = {
...
+		getDescription: () => {},
+	},
+	[ FEATURE_EDGE_CACHING_V2 ]: {
+		getSlug: () => FEATURE_EDGE_CACHING_V2,
+		getTitle: () => 'High-performance edge caching',
+		getDescription: () => {},
+	},
+	[ FEATURE_ADVANCED_SEO_EXPANDED_ABBR_V2 ]: {
+		getSlug: () => FEATURE_ADVANCED_SEO_EXPANDED_ABBR,
+		getTitle: () => i18n.translate( 'Advanced Search Engine Optimisation (SEO) tools' ),
```
Comment (donlair) [inline client/lib/plans/features-list.js]: Change to `Advances SEO tools`

**Review by donlair — CHANGES_REQUESTED** [review]
I added some requests for copy tweaks but the PR works as expected. I think this is ready to launch once those updates are in-place.

File: client/my-sites/plan-features-comparison/index.jsx (hunk 1 above)
Comment (niranjan-uma-shankar) [reply client/my-sites/plan-features-comparison/index.jsx]: Fixed.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7512076

Thank you @niranjan-uma-shankar for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7512076).

**Pull request merged**